djstripe/signals.py

```diff
@@ -236,16 +250,18 @@
     "treasury.outbound_payment.failed",
     "treasury.outbound_payment.posted",
     "treasury.outbound_payment.returned",
+    "treasury.outbound_payment.tracking_details_updated",
     "treasury.outbound_transfer.canceled",
     "treasury.outbound_transfer.created",
     "treasury.outbound_transfer.expected_arrival_date_updated",
     "treasury.outbound_transfer.failed",
     "treasury.outbound_transfer.posted",
     "treasury.outbound_transfer.returned",
+    "treasury.outbound_transfer.tracking_details_updated",
     "treasury.received_credit.created",
     "treasury.received_credit.failed",
     "treasury.received_credit.succeeded",
-    "treasury.received_debit.created",
+    "treasury.received_debit.created"
```

Comment:
This is wrong and will merge the string with the ping below.

Reply:
good catch
Copied it from the stripe json spec

Either way removed it for now form this PR